### DISCUSSION TITLE
How to link an image in the Bard editor?

### DISCUSSION THREAD

**Discussion by anakadote** [statamic/cms#5788] [Troubleshooting]
Is it possible to wrap an image with a link in the Bard editor without a using a set? I've tried selecting the image then clicking the link icon, but no dice. Seems like this would be a common thing to do/want.

**Comment by afonic** [comment]
As far as I know you can't. Using a set is the recommended way to do anything complex in Bard.

Feel free to open a FR: https://github.com/statamic/cms
